.devcontainer/Dockerfile

```diff
@@ -21,19 +21,11 @@ ENV PATH=$PATH:/home/vscode/.dapr/bin
 
 ENV DOCKER_BUILDKIT=1
 
-# Fix for bug in devcontainer features
-ADD https://raw.githubusercontent.com/devcontainers/features/7fa90110d762797cc0b1c2fe8fcc028c9b813d56/src/common-utils/install.sh /tmp/scripts/install-common.sh
-RUN UID="4000" bash /tmp/scripts/install-common.sh
-ADD https://raw.githubusercontent.com/devcontainers/features/7fa90110d762797cc0b1c2fe8fcc028c9b813d56/src/docker-in-docker/install.sh /tmp/scripts/install-dind.sh
-RUN VERSION="20.10" bash /tmp/scripts/install-dind.sh
```

Comment:
I see that this is also removed in the template, but I am not sure if this is a good idea, since there might be some version mismatch between this repo and the baseimage.

If we agree that the baseimage is always correct, we could also remove the installation of the python dependencies.

Reply:
That is removed, because we decided to use native (officially supported) way of features installation, and now base image already contains features, it means that base image is always correct.

Reply:
I mean we still have the problem if want to have different versions of the features, but for now it is ok...